crates/turborepo-lib/src/execution_state.rs

```diff
@@ -28,11 +28,10 @@ impl<'a> TryFrom<&'a CommandBase> for ExecutionState<'a> {
     type Error = anyhow::Error;
 
     fn try_from(base: &'a CommandBase) -> Result<Self, Self::Error> {
-        let root_package_json =
-            PackageJson::load(&base.repo_root.join_component("package.json")).ok();
+        let root_package_json = PackageJson::load(&base.repo_root.join_component("package.json"))?;
```

Comment:
Hmm I don't remember why I made this change. Will revert, although I do think we need to rethink our package manager inference a little post-port because silently failing on package.json parsing can cause some issues.